### PR TITLE
Use kDefaultGeneralSupportHeight throughout

### PR DESCRIPTION
**Pull request by Gymnasiast**
Split off from #22035

### CODE REVIEW

File: src/openrct2/ride/coaster/InvertedRollerCoaster.cpp [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #include "../../paint/tile_element/Paint.TileElement.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/coaster/InvertedRollerCoaster.cpp:17]: This file doesn't seem to use the new constant.

File: src/openrct2/ride/coaster/SuspendedSwingingCoaster.cpp [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include "../../paint/tile_element/Paint.TileElement.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/coaster/SuspendedSwingingCoaster.cpp:17]: Default support height not used in this file

File: src/openrct2/ride/coaster/WoodenRollerCoaster.h [hunk 3]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/coaster/WoodenRollerCoaster.h:18]: Default support height not used in this file

**Review by AaronVanGeffen — COMMENTED** [review]
The `Support.h` include seems to have been added to quite a few files that don't appear to use it, e.g. the gentle rides. That should probably be corrected.

File: src/openrct2/ride/gentle/Circus.cpp [hunk 4]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/Circus.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/gentle/CrookedHouse.cpp [hunk 5]
```diff
@@ -14,6 +14,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/CrookedHouse.cpp:17]: Default support height not used in this file

File: src/openrct2/ride/gentle/Dodgems.cpp [hunk 6]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/Dodgems.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/gentle/FerrisWheel.cpp [hunk 7]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/FerrisWheel.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/gentle/FlyingSaucers.cpp [hunk 8]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/FlyingSaucers.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/gentle/HauntedHouse.cpp [hunk 9]
```diff
@@ -14,6 +14,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/HauntedHouse.cpp:17]: Default support height not used in this file

File: src/openrct2/ride/gentle/MerryGoRound.cpp [hunk 10]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/MerryGoRound.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/gentle/SpaceRings.cpp [hunk 11]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/SpaceRings.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/gentle/SpiralSlide.cpp [hunk 12]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/gentle/SpiralSlide.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/thrill/3dCinema.cpp [hunk 13]
```diff
@@ -13,6 +13,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/3dCinema.cpp:16]: Default support height not used in this file

File: src/openrct2/ride/thrill/Enterprise.cpp [hunk 14]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/Enterprise.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/thrill/MagicCarpet.cpp [hunk 15]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/MagicCarpet.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/thrill/MotionSimulator.cpp [hunk 16]
```diff
@@ -14,6 +14,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/MotionSimulator.cpp:17]: Default support height not used in this file

File: src/openrct2/ride/thrill/SwingingInverterShip.cpp [hunk 17]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/SwingingInverterShip.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/thrill/SwingingShip.cpp [hunk 18]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/SwingingShip.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/thrill/TopSpin.cpp [hunk 19]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/TopSpin.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/thrill/Twist.cpp [hunk 20]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/support/WoodenSupports.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/thrill/Twist.cpp:18]: Default support height not used in this file

File: src/openrct2/ride/transport/SuspendedMonorail.cpp [hunk 21]
```diff
@@ -15,6 +15,7 @@
 #include "../../paint/tile_element/Paint.TileElement.h"
 #include "../../paint/tile_element/Segment.h"
 #include "../../paint/track/Segment.h"
+#include "../../paint/track/Support.h"
```
Comment (spacek531) [inline src/openrct2/ride/transport/SuspendedMonorail.cpp:18]: Default support height not used in this file

**Review by spacek531 — CHANGES_REQUESTED** [review]
Everything looks good. A few files received the include which don't need it. When the extra includes are removed, this is good 👍

**Review by AaronVanGeffen — APPROVED** [review]

**Pull request merged**